src/Command/IndexService/ResetQueueCommand.php

```diff
@@ -26,6 +27,11 @@
 /**
  * @internal
  */
+#[AsCommand(
+    name: 'ecommerce:index:reset-indices',
```

Comment:
```suggestion
    name: 'ecommerce:index:reset-all',
```
?
same as above (docs)